docs/patterns/observability/logs.md

```diff
@@ -57,3 +57,24 @@ Once logs have been sent to Log Analytics, that can also be consumed by Applicat
 
 You can now interact with your logs in either of these instances.
 
+
+### Configuring Outbound Proxy Support for Fluent Bit
+If you require outbound proxy support:
+
+1. In `values.yaml`:
+    
+    a) Change `enableProxySupport` to `true`
+
+    b) Update the httpProxy and httpsProxy values to `"http://<host>:<port>"`
+
+2. In `/charts/osm/templates` create a new secret `proxy-config.yaml`:
```

Comment:
This proxy-config.yaml can be created as part of the chart conditional on `enableProxySupport`